app/models/request.rb

```diff
@@ -19,9 +19,7 @@ class Request < ApplicationRecord
   belongs_to :distribution, optional: true
   has_many :item_requests, dependent: :destroy
 
-  scope :active, -> { where(status: "Active") }
-
-  STATUSES = %w[Active Fulfilled].freeze
+  enum status: { pending: 0, started: 1, fulfilled: 2 }, _prefix: true
```

Comment:
I love the change to an enum. I think on `app/views/requests/_request_row.html.erb` we're going to need to update that partial so the requests don't come in lower case. I think instead of `request_row.status` we do something like `request_row.status.humanize`.

Reply:
Good point. I'll change that. Thank you

Reply:
+1 for changing it to an enum. I think there's an issue created for that, I'll add a closure to the description.  :100: